backend/main/docker-compose.yml

```diff
@@ -5,9 +5,10 @@ services:
     volumes:
       - .:/code
     working_dir: /code
+    command: bash -c "npm ci && npm run test:run"
     depends_on:
-      - dynamo-db
-      - initializer
+      initializer:
+        condition: service_completed_successfully
```

Comment:
This condition means that this container waits until the initializing Table is complete.

https://docs.docker.com/compose/compose-file/05-services/#long-syntax-1